exp/views/study.py

```diff
@@ -910,53 +914,60 @@ def get_context_data(self, **kwargs):
         # two jobs - generate statistics and populate k/v store.
         for response in responses:
 
-            stat_for_status = statistics.get(response.most_recent_ruling)
+            stat_for_status = statistics.get(response["current_ruling"])
             stat_for_status["responses"] += 1
-            stat_for_status["children"].add(response.child)
+            stat_for_status["children"].add(response["child_id"])
             total_stats["responses"] += 1
-            total_stats["children"].add(response.child)
+            total_stats["children"].add(response["child_id"])
 
-            response_data = response_key_value_store[str(response.uuid)] = {}
+            response_json = response_key_value_store[str(response["uuid"])] = {}
 
-            response_data["videos"] = [
+            response_json["videos"] = [
                 {"aws_url": video.download_url, "filename": video.filename}
-                for video in response.consent_videos
+                for video in response["videos"]
```

Comment:
it looks like response["videos"] has only the consent videos due to how get_responses_with_current_rulings_and_videos works - confirm?

Reply:
confirmed